java/README.md

```diff
@@ -238,3 +273,52 @@ IMPORTANT: Please note that the gradle build script included in that directory a
     </plugins>
   </build>
 ```
+
+## Limitations
+
+Currently, Piranha does not distinguish between enum usages for a feature flag to be cleaned up, versus for another, non-feature flag usage.
```

Comment:
Given Piranha removes the method call, once it is configured, I am not sure whether this merits a discussion in the README. This is expected behavior even for non-enum usages where the method cleanup will not happen if the method is not configured. Can we get rid of this discussion?

Reply:
Yeah wasn't sure if it was helpful, removed. I purposefully put it in its own commit though so it was easy to remove 😂